js/notify.js

```diff
@@ -15,7 +15,7 @@ define([
             level: type == 'error' ? 'danger' : type,
             message: message
         }));
-        Log.debug('Notify:', {type: type, message: message, container: container});
+        Log.default.debug('Notify:', {type: type, message: message, container: container});
```

Comment:
Well, I think that turning that file to ES6 would allow you to use directly `import log from 'logger';` and that's the way to go ;)